config/environments/production.rb

```diff
@@ -81,9 +81,7 @@
   # Do not dump schema after migrations.
   config.active_record.dump_schema_after_migration = false
 
-  # See https://github.com/redis-store/redis-rails#usage
-  config.cache_store = :redis_store, "#{Rails.application.config_for(:cable)['url']}/cache",
-                       { expires_in: 90.minutes }
+  config.cache_store = :redis_cache_store, { url: Rails.application.config_for(:cable)['url'] }
```

Comment:
Deze url eindigde eerst mer `/cache`. Dit is niet meer nodig?

Reply:
Zover ik online kan zien niet